README.zh-CN.md

```diff
@@ -117,104 +107,31 @@ bash bin/stop.sh
-| `eventMesh.server.grpc.port`       | `10205` | `EventMesh grpc server port` | 
-| `eventMesh.server.admin.http.port` | `10106` | `HTTP management port`       | 
-
-#### 4.配置 `rocketmq-client.properties`
-
-这个配置文件中包含 EventMesh 运行时环境和集成进来的其他插件所需的参数。
-
-下载配置文件（替换下载链接中的 `1.10.0` 为您正在使用的版本）
-```
-sudo wget https://raw.githubusercontent.com/apache/eventmesh/1.10.0-prepare/eventmesh-storage-plugin/eventmesh-storage-rocketmq/src/main/resources/rocketmq-client.properties
-```
-
-使用下面的 `vim` 命令编辑 `rocketmq-client.properties`。
-```
-sudo vim rocketmq-client.properties
-```
-
-> 请注意，如果您正在运行的 namesrv 地址不是配置文件中的默认值，请将其修改为实际正在运行的 namesrv 地址。
-
-请检查配置文件里的默认namesrvAddr是否已被占用，如果被占用请修改成未被占用的地址:
-
-| 属性                                      | 默认值                             | 备注                                 |   
-|-----------------------------------------|---------------------------------|------------------------------------|  
-| `eventMesh.server.rocketmq.namesrvAddr` | `127.0.0.1:9876;127.0.0.1:9876` | `RocketMQ namesrv default address` |
-
-#### 5.运行EventMesh
+#### 2. 运行EventMesh
 
 现在你就可以开始根据下载好的EventMesh镜像运行容器了。
 
-使用到的命令是 `docker run`，有以下两点内容需要格外注意。
+使用到的命令是 `docker run`。
+
 - 绑定容器端口和宿主机端口: 使用 `docker run` 的 `-p` 选项。
-- 将宿主机中的两份配置文件挂在到容器中: 使用 `docker run` 的 `-v` 选项。
 
 综合一下，对应的启动命令为:
 ```
-sudo docker run -d --name eventmesh -p 10000:10000 -p 10105:10105 -p 10205:10205 -p 10106:10106 -v /data/eventmesh/conf/eventmesh.properties:/data/app/eventmesh/conf/eventmesh.properties -v /data/eventmesh/conf/rocketmq-client.properties:/data/app/eventmesh/conf/rocketmq-client.properties -t apache/eventmesh:latest
-```
-
-接下来，你可以使用下面的命令查看容器的状态。
-```
-sudo docker ps
+sudo docker run -d --name eventmesh -p 10000:10000 -p 10105:10105 -p 10205:10205 -p 10106:10106 -t apache/eventmesh:latest
 ```
 
 如果成功的话，你会看到终端打印出了如下所示容器的信息，其中就有运行 EventMesh 镜像的容器。
 ```
 $ sudo docker ps
-CONTAINER ID   IMAGE                        COMMAND                  CREATED         STATUS         PORTS                                                                                                                                                                 NAMES
-5bb6b6092672   apache/eventmesh:latest      "/bin/sh -c 'sh star…"   6 seconds ago   Up 3 seconds   0.0.0.0:10000->10000/tcp, :::10000->10000/tcp, 0.0.0.0:10105-10106->10105-10106/tcp, :::10105-10106->10105-10106/tcp, 0.0.0.0:10205->10205/tcp, :::10205->10205/tcp   eventmesh
+CONTAINER ID   IMAGE                     COMMAND                   CREATED         STATUS         PORTS                                                                                                                                  NAMES
+9c08130ee797   apache/eventmesh:latest   "bash bin/start.sh"       9 seconds ago   Up 8 seconds   0.0.0.0:10000->10000/tcp, 0.0.0.0:10105-10106->10105-10106/tcp, 0.0.0.0:10205->10205/tcp                                               eventmesh
 ```
 
 读取 EventMesh 容器的日志：
```

Comment:
``````suggestion
使用以下命令启动 EventMesh 容器：

```shell
sudo docker run -d --name eventmesh -p 10000:10000 -p 10105:10105 -p 10205:10205 -p 10106:10106 -t apache/eventmesh:latest
```

进入容器：

```shell
sudo docker exec -it eventmesh /bin/bash
```

查看日志：
``````

'docker exec' command missing here. Please refer to the updated version of the documentation.

Reply:
Please delete from `现在你就可以开始根据下载好的EventMesh镜像运行容器了` to `读取 EventMesh 容器的日志：`, and add the seggested version.

Reply:
> Reference in
Done

Reply:
It seems they are still there:

![image](https://github.com/apache/eventmesh/assets/41445332/83d9305f-0d0c-4b8e-9861-7fdb04dbed53)

Reply:
Got it, done.

Reply:
You still haven't followed the narrative I provided in the suggestion. Allow me to explain the reasons.

The phrase `查看 EventMesh 容器的日志` is incorrect. The command to view Docker container logs is `docker logs`, not `tail -f`. Therefore, we should not use the phrase `查看 EventMesh 容器的日志`. Instead, we simply use "查看日志" and that's enough.

Please remove `（将 eventmesh 替换为您指定的容器名称或 ID）` because we did not inform the user that `eventmesh` is a customizable container name when creating the container. Therefore, when the user reads this, they will be confused: "Do we need to modify it? If yes, what should we modify it to?" Let's avoid causing confusion for the users.

Reply:
OK